docs/source/contributing.rst

```diff
@@ -36,28 +36,24 @@ Wagtailtrans made it very easy to setup a runnable Django project to help with t
 
 * We suggest you use a virtual environment for development.
 
-* Activate your virtual environment and install following packages.
+* Set PostgreSQL Database Authentication parameters host, user, and password as environment variables.
```

Comment:
In development use-cases it is probably relevant to just have a database name and a postgres user as a default.